go.mod

```diff
@@ -15,7 +15,8 @@ require (
 	github.com/google/uuid v1.6.0
 	github.com/onsi/gomega v1.34.1
 	github.com/open-policy-agent/cert-controller v0.11.0
-	github.com/open-policy-agent/frameworks/constraint v0.0.0-20240927180816-0f64229c5539
+	github.com/open-policy-agent/frameworks/constraint v0.0.0-20241007142041-e84361fed758
```

Comment:
@abhipatnala this should point to [e78c8abd754aa12538eb21ee0c145cbe5465569d](https://github.com/open-policy-agent/frameworks/commit/e78c8abd754aa12538eb21ee0c145cbe5465569d)